impl/src/planning/subplanners.rs

```diff
@@ -143,40 +134,44 @@ impl<'planner> WorkspaceSubplanner<'planner> {
       workspace_path: self.settings.workspace_path.clone(),
       gen_workspace_prefix: self.settings.gen_workspace_prefix.clone(),
       output_buildfile_suffix: self.settings.output_buildfile_suffix.clone(),
+      workspace_members: self
+        .metadata
+        .metadata
+        .workspace_members
+        .iter()
+        .filter_map(|pkg_id| {
+          if let Some(pkg) = self
+            .metadata
+            .metadata
+            .packages
+            .iter()
+            .find(|pkg| pkg.id == *pkg_id)
```

Comment:
Nit: Made this comment elsewhere, duplicating here.

Can we avoid these extremely long conditional clauses?

```rust
let workspace_member = self
            .metadata
            .metadata
            .packages
            .iter()
            .find(|pkg| pkg.id == *pkg_id);
if let Some(workspace_member) = workspace_member {
```


```

Reply:
I'll keep this in mind for future changes.